core/src/main/java/org/verapdf/pdfa/package-info.java

```diff
@@ -24,11 +24,9 @@
  * Developers wishing to integrate veraPDF PDF/A validation into
  * their own Java applications should only need to use:
  * <ul>
- * <li>{@link org.verapdf.pdfa.ValidatorFactory} to obtain a {@link org.verapdf.pdfa.Validator} instance.</li>
```

Comment:
If these classes moved, you need to change packages.

Reply:
returned and fixed